man/compute_dtf.Rd

```diff
@@ -1,5 +1,5 @@
 % Generated by roxygen2: do not edit by hand
-% Please edit documentation in R/derive_date_vars.R
+% Please edit documentation in R/derive_vars_dt.R
```

Comment:
> Why have so many files `.Rd` files changed?

The migration caused at least these changes, as for the others I'm not really sure, just ran `devtool::document()` like usual